### ISSUE TITLE
CSS Custom Properties in var () with fallback value are not replaced

### ISSUE THREAD

**Issue by klimashkin** [JPeer264/node-rename-css-selectors#42]
Hi! Seems like if there is a fallback in `var()`, then property name doesn't get replaced, like here:
```css
border: var(--button-border-width, var(--border-width)) 
        solid 
        var(--button-border-color, transparent);
```

**Comment by klimashkin** [comment]
Btw, I think `3.2.3` has not been published : )

**Comment by JPeer264** [comment]
Hey sorry, I was on holiday. Published now

**Comment by JPeer264** [comment]
Oh fallbacks. Ok I will work on those.

**Comment by JPeer264** [comment]
Resolved in v`3.2.4` published as well.
